app/src/NativeBase/Components/Forms/DatePicker.tsx

```diff
@@ -0,0 +1,195 @@
+/**
+ * @file DatePicker.
+ */
+
+import React, { FC, useState } from 'react'
+import CalendarPicker from 'react-native-calendar-picker'
+import { Pressable, Dimensions } from 'react-native'
+import { VStack, Box, Text, Input, useColorMode, Icon } from 'native-base'
+import dayjs from 'dayjs'
+import StaTheme from '../../Theme/StaTheme'
+import MaterialIcons from 'react-native-vector-icons/MaterialIcons'
+
+interface DatePickerProps {
+  description: string
+  maximumDate?: Date
+  minimumDate?: Date
+  onChange: (date: Date) => void
+  value: Date
+}
+
+const DatePicker: FC<DatePickerProps> = ({
+  description,
+  maximumDate,
+  minimumDate,
+  onChange,
+  value,
+}) => {
+  const [pickerActive, setPickerActive] = useState(false)
+  const [selectedStartDate, setSelectedStartDate] = useState(null)
+  const [todaySelected, setTodaySelected] = useState(false)
+  const { colorMode } = useColorMode()
+  const { width } = Dimensions.get('window')
+
+  const handleChange = (value: Date) => {
+    onChange(value)
+    setSelectedStartDate(value)
+    setPickerActive(false)
+    if (dayjs(value).format('DD/MM/YY') === dayjs().format('DD/MM/YY')) {
+      setTodaySelected(true)
+    } else {
+      setTodaySelected(false)
+    }
+  }
+
+  const customDatesStylesCallback = (date: Date) => {
```

Comment:
Small thing but might be nice to have a comment here explaining briefly what this is doing -- is it to highlight the date if it's today (or something else)?